_pkgdown.yml

```diff
@@ -6,6 +6,7 @@ reference:
 
   - title: Main functions
     contents:
+      - starts_with("bar_")
```

Comment:
@maurolepore Thanks for your answer! This effectively removed the failing check(). :)